src/components/forms/Admin.jsx

```diff
@@ -1,127 +1,26 @@
 "use client";
 
 import { useState } from "react";
-import { USER } from "../../data/forms/User";
-import Radio from "@/components/Radio";
-import Checkbox from "../Checkbox";
-import Input from "../Input";
-import Col from "react-bootstrap/Col";
-import Row from "react-bootstrap/Row";
-import Button from "../Button";
-import { AFFILIATIONS, POSITIONS, REQUIREMENTS } from "@/data/forms/Admin";
+import { HELPER } from "../../data/forms/Admin.js";
+import Form from "@/app/forms/Form.jsx";
+import { FIELDS } from "../../data/forms/Admin.js";
+import { useSession } from "next-auth/react";
 
 const Admin = () => {
-  const [user, setUser] = useState(USER);
-  const [requirements, setRequirements] = useState(REQUIREMENTS);
-
-  const handleSubmit = () => {
-    console.log(user);
-    console.log(requirements);
-  };
-
-  const handleRequirementsCheckbox = (key, value) => {
-    setRequirements({
-      ...requirements,
-      [key]: { state: !value.state, text: value.text },
-    });
-  };
-
+  const { data: session } = useSession();
+  const [admin, setAdmin] = useState({
+    name: session.user.name,
```

Comment:
this info should be collected in the end, otherwise we store it and do nothing until the end, check volunteer